.gitignore

```diff
@@ -6,6 +6,7 @@ npm-debug.log
 yarn-error.log
 yarn.lock
 /benchmark/
+/performance-tests/*/*
```

Comment:
create a working directory in your test and ignore it only

Reply:
Done.